terratest/test/go.mod

```diff
@@ -0,0 +1,10 @@
+module testterratest
+
+go 1.14
```

Comment:
- It is new project; start  with GO 1.15 (DockerFile, mod files)

Reply:
done